docs/cimpl6Authoring.md

```diff
@@ -1,6 +1,4 @@
-# CIMPL Authoring Guide
-
->**Note:** This documentation is in the process of being further expanded and is an actively updating document.
```

Comment:
Keep this note -- this is very much a preliminary document and the user should be aware.

Reply:
OK. Slight wording change.